### ISSUE TITLE
Array of Trail Lines

### ISSUE THREAD

**Issue by cassiussa** [cassiussa/startography#44]
The Trail Line gameObjects (planets, moons?) should be put into an array that is static.  Assign values, such as whether or not it is active in the view.  This way we can also check the distance from the planet to the camera to decide on the size (visibility) of the trailing line from the camera's perspective.  And because we can easily iterate over any number of items from the array per Update(), we should be able to easily streamline the processing requirements.

For example, if we have 5,000 items and want to check the distance from the camera to the planet so we can scale its trail, we'd check if the item is active.  When it hits the first result, we can then update its scale.  Then return so that no further processing occurs this Update().  When the next Update() occurs, we'd continue on to the next item that is active, change it's trailing scale, and repeat.

So we're only performing a series of boolean operation checks and then performing processing on just 1 distance check per frame.  Distance checking is expensive.

**Comment by cassiussa** [comment]
I have added an Items.cs script, which is global and resides on the "Globals" gameObject.  Additionally, I've added a RegisterItem.cs script which should be added, at startup, to each object that is of Type Star, Planet, or Moon.  It should be the last item added to any given object as it will need all items to be set up already so that it can add those details into Lists within the Items.cs script.  We can than more easily access any item or component later by iterating over the applicable List.
